ai-knowledge/seo-optimization.md

```diff
@@ -0,0 +1,92 @@
+# SEO Keyword Optimization Guide
```

Comment:
do you need both of these? I think you need to migrate the astro related instructions possibly into here, and that's it.